sdk/storage/Azure.Storage.Common/src/Constants.cs

```diff
@@ -427,5 +427,16 @@ internal static class Xml
             internal const string Code = "Code";
             internal const string Message = "Message";
         }
+
+        internal static class GeoRedundantReadConstants
```

Comment:
Drop the suffix `Constants`.  You'll access this via `Constants.*` so it's redundant.